src/cpp/fastdds/domain/DomainParticipantImpl.hpp

```diff
@@ -85,7 +85,7 @@ class DomainParticipantImpl
 
 public:
 
-    ReturnCode_t enable();
+    virtual ReturnCode_t enable();
```

Comment:
I don't think `virtual` is needed here
```suggestion
    ReturnCode_t enable();
```